test/unit/parse-siti-payment-file/parse-payment-file.test.js

```diff
@@ -34,62 +38,62 @@ describe('Parse and send events on success or failure', () => {
   })
 
   test('should call buildAndTransformParseFile when valid filename, fileBuffer and sequence are received', async () => {
-    await parsePaymentFile(filename, fileBuffer, sequence)
+    await parsePaymentFile(filename, fileBuffer, schemeType)
     expect(buildAndTransformParseFile).toHaveBeenCalled()
   })
 
   test('should call buildAndTransformParseFile with fileBuffer and sequence when valid filename, fileBuffer and sequence are received', async () => {
-    await parsePaymentFile(filename, fileBuffer, sequence)
-    expect(buildAndTransformParseFile).toHaveBeenCalledWith(fileBuffer, sequence)
+    await parsePaymentFile(filename, fileBuffer, schemeType)
+    expect(buildAndTransformParseFile).toHaveBeenCalledWith(fileBuffer, schemeType)
   })
 
   test('should call buildAndTransformParseFile when invalid filename, fileBuffer and sequence are received', async () => {
     filename = ''
     fileBuffer = ''
-    sequence = ''
+    schemeType = ''
 
-    await parsePaymentFile(filename, fileBuffer, sequence)
+    await parsePaymentFile(filename, fileBuffer, schemeType)
     expect(buildAndTransformParseFile).toHaveBeenCalled()
   })
 
   test('should call buildAndTransformParseFile with fileBuffer and sequence when invalid filename, fileBuffer and sequence are received', async () => {
     filename = ''
     fileBuffer = ''
-    sequence = ''
+    schemeType = ''
 
-    await parsePaymentFile(filename, fileBuffer, sequence)
-    expect(buildAndTransformParseFile).toHaveBeenCalledWith(fileBuffer, sequence)
+    await parsePaymentFile(filename, fileBuffer, schemeType)
+    expect(buildAndTransformParseFile).toHaveBeenCalledWith(fileBuffer, schemeType)
   })
 
   test('should call sendBatchProcessedEvents when valid filename, fileBuffer and sequence are received', async () => {
-    await parsePaymentFile(filename, fileBuffer, sequence)
+    await parsePaymentFile(filename, fileBuffer, schemeType)
     expect(sendBatchProcessedEvents).toHaveBeenCalled()
   })
 
   test('should call sendBatchProcessedEvents with filename, paymentRequests and sequence when valid filename, fileBuffer and sequence are received', async () => {
-    await parsePaymentFile(filename, fileBuffer, sequence)
-    expect(sendBatchProcessedEvents).toHaveBeenCalledWith(filename, mockPaymentRequests, sequence, '2021-08-12')
+    await parsePaymentFile(filename, fileBuffer, schemeType)
+    expect(sendBatchProcessedEvents).toHaveBeenCalledWith(filename, mockPaymentRequests, schemeType.batchId, '2021-08-12')
```

Comment:
Think this date is gotten from the `filename`, could you make it more explicit by creating a variable and using it?